src/rcdgen/check_data.cpp

```diff
@@ -2810,6 +2812,15 @@ static std::shared_ptr<INFOBlock> ConvertINFONode(std::shared_ptr<NodeGroup> ng)
 
 	char buffer[32];
 	time_t cur_time = time(nullptr);
+	char *source_date_epoch = std::getenv("SOURCE_DATE_EPOCH");
+	if (source_date_epoch) {
+		std::istringstream iss(source_date_epoch);
+		iss >> cur_time;
+		if (iss.fail() || !iss.eof()) {
+			fprintf(stderr, "Error: Cannot parse SOURCE_DATE_EPOCH as integer\n");
+			exit(27);
```

Comment:
Not sure this should be a fatal error, perhaps just print the warning and fall back to current time?